dashboards/migrations/0005_auto_20170308_0831.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.12 on 2017-03-08 08:31
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('dashboards', '0004_definition'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='definition',
+            options={},
+        ),
+    ]
```

Comment:
It was generated when I ran makemigrations for the field that I added, not sure where it comes from. Should I remove it from this PR?